transport/processing/processors.py

```diff
@@ -0,0 +1,48 @@
+import abc
+import functools
+import os
+
+import container.model
+import container.util
+
+OWN_DIR = os.path.abspath(os.path.dirname(__file__))
+
+
+class ProcessorBase():
```

Comment:
nit: parantheses are optional in python3 (-> `class ProcessorBase:`)

Reply:
great, I thought only `object` was not required anymore